packages/cli/src/commands/ts-artifacts.ts

```diff
@@ -151,15 +197,17 @@ export async function generateTsArtifacts({
             `import { MeshContext as BaseMeshContext, MeshInstance } from '@graphql-mesh/runtime';`,
             `import { InContextSdkMethod } from '@graphql-mesh/types';`,
           ];
+          const sourceTypeDeclarationImports = sourceTypeDeclarations.map(d => d.importStatement);
           const sdkItems: string[] = [];
           const contextItems: string[] = [];
           const results = await Promise.all(
             rawSources.map(source => {
-              const sourceMap = unifiedSchema.extensions.sourceMap as Map<RawSourceOutput, GraphQLSchema>;
-              const sourceSchema = sourceMap.get(source);
+              const { normalizedSourceName } = sourceTypeDeclarations.find(
+                ({ sourceName }) => sourceName === source.name
+              )!;
               const item = generateTypesForApi({
-                schema: sourceSchema,
-                name: source.name,
+                schema: source.schema,
```

Comment:
This is where I've switched the context sdks to be built against the untransformed upstream schemas.

Reply:
InContext SDK calls `delegateToSchema` of GraphQL Tools against transformed schema. In this PR's scope, we need to follow it. We can discuss having different SDKs for other use cases in a different discussion or PRs later :)